api/src/controllers/publications.js

```diff
@@ -143,21 +139,20 @@ function readAllPublicationsByTeam(req, res, next) {
  */
 async function getGoogleScholarPublications(req, res) {
   const author = req.params.gScholarUserId;
-  const startFrom = req.params.startFrom;
+  const { startFrom } = req.params;
   const teamId = req.params.team_id;
 
-  const noOfDummyLinks = playwrightConfig.noOfDummyLinks;
-  const pageSize = playwrightConfig.pageSize;
-  const url =
-    playwrightConfig.baseUrl +
-    author +
-    playwrightConfig.startSuffix +
-    startFrom +
-    playwrightConfig.pageSizeSuffix +
-    pageSize +
-    playwrightConfig.sortBySuffix;
+  const { noOfDummyLinks } = playwrightConfig;
+  const { pageSize } = playwrightConfig;
+  const url = playwrightConfig.baseUrl
+    + author
+    + playwrightConfig.startSuffix
+    + startFrom
+    + playwrightConfig.pageSizeSuffix
+    + pageSize
+    + playwrightConfig.sortBySuffix;
```

Comment:
Another thing I noticed is that the AirBnB style guide conflicts with prettier sometimes and this block is an example.  Can we somehow configure prettier to follow this style if we're going to be enforcing it?

Reply:
Not too sure, but for VSCode users, you can enable this extension https://daveceddia.com/vscode-use-eslintrc/#:~:text=Configure%20VSCode%20Settings%20to%20use%20ESLint%20for%20Formatting&text=Click%20that%20tiny%20icon%20in,paper%20with%20a%20little%20arrow.&text=The%20first%20one%20turns%20on,it%2C%20we're%20done.

Similar exists for Webstorm users. And re: prettier, we may need someone to look into that,